src/libs/antares/series/series.cpp

```diff
@@ -74,24 +72,17 @@ int TimeSeries::saveToFile(const std::string& filename, bool saveEvenIfAllZero)
 
 double TimeSeries::getCoefficient(uint32_t year, uint32_t timestep) const
 {
-    if (timeSeries.width == 0)
-        return 0;
     return timeSeries[getSeriesIndex(year)][timestep];
 }
 
 const double* TimeSeries::getColumn(uint32_t year) const
 {
-    if (timeSeries.width == 0)
-        return emptyColumn.data();
     return timeSeries[getSeriesIndex(year)];
 }
 
 uint32_t TimeSeries::getSeriesIndex(uint32_t year) const
 {
-    if (timeSeries.width == 1)
```

Comment:
**Explanation** : now, at construction of **timeseriesNumbers**, default content is **zero** for every year, that's why we don't need the previous complexity.